feature_engineering/test_feature_engineering.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
```

Comment:
Why not break down this test module into isolated Unit Test cases? It would help with several things:
  1. Make sure each test case is isolated
  2. Make sure the test is maintainable in the future

Guiding example: https://docs.python.org/3/library/unittest.html#basic-example

Reply:
updated